src/components/parameters/Parameters.svelte

```diff
@@ -24,51 +25,74 @@
   $: sortedFormParameters = formParameters.sort((a, b) => compare(a.order, b.order));
 </script>
 
-{#each sortedFormParameters as formParameter (formParameter.name)}
-  <Highlight highlight={highlightKeysMap[formParameter.name]}>
-    <div bind:clientWidth class="parameter">
-      {#if formParameter.schema.type === 'series' || formParameter.schema.type === 'struct'}
-        <ParameterRec
-          {disabled}
-          {expanded}
-          {formParameter}
-          {hideRightAdornments}
-          {labelColumnWidth}
-          {level}
-          {levelPadding}
-          {parameterType}
-          on:change
-          on:reset
-          {use}
-        />
-      {:else}
-        <ParameterBase
-          {disabled}
-          {formParameter}
-          {hideRightAdornments}
-          {labelColumnWidth}
-          {level}
-          {levelPadding}
-          {parameterType}
-          on:change
-          on:reset
-          {use}
-        />
-      {/if}
-    </div>
-  </Highlight>
-{/each}
+<div class="parameters-container">
+  {#each sortedFormParameters as formParameter (formParameter.name)}
+    <Highlight highlight={highlightKeysMap[formParameter.name]}>
+      <div bind:clientWidth class="parameter">
+        {#if formParameter.schema.type === 'series' || formParameter.schema.type === 'struct'}
+          <ParameterRec
+            {disabled}
+            {expanded}
+            {formParameter}
+            {hideRightAdornments}
+            {labelColumnWidth}
+            {level}
+            {levelPadding}
+            {parameterType}
+            on:change
+            on:reset
+            {use}
+          />
+        {:else}
+          <ParameterBase
+            {disabled}
+            {formParameter}
+            {hideRightAdornments}
+            {labelColumnWidth}
+            {level}
+            {levelPadding}
+            {parameterType}
+            on:change
+            on:reset
+            {use}
+          />
+        {/if}
+        <div class="parameter-info">
+          <ParameterInfo {formParameter} on:reset />
+        </div>
+      </div>
+    </Highlight>
+  {/each}
+</div>
 
 <style>
   .parameter {
-    display: flex;
-    flex-direction: column;
-    gap: 4px;
-    padding: 4px 0px;
+    column-gap: 4px;
+    display: grid;
+    grid-template-columns: auto 16px;
   }
 
   .parameter :global(.st-input.error) {
     background-color: inherit;
     color: inherit;
   }
+
+  .parameters-container :global(> div.highlight) {
+    box-sizing: border-box;
+    padding: 4px 0;
+  }
+
+  .parameters-container :global(> div.highlight:hover) {
+    border-bottom: 1px solid var(--st-gray-20, #ebecec);
```

Comment:
Using a border here introduces a bit of a jiggle. Could use a box shadow instead which doesn't affect element size? Something like:
```
box-shadow: 0px 0px 0px 1px var(--st-gray-20);
```
Wouldn't need to play with the padding anymore either?

Reply:
hmm...i'm not seeing the jiggle, but your solution gives the same effect for me so I'll go with it :+1:

Reply:
More apparent when you have parameters with toggles on the left

Reply:
Oh and maybe when you're zoomed in on the page with cmd/ctrl +/-?